src/main/webapp/app/components/sidebar/GenericSidebar.tsx

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+import './generic-sidebar.scss';
+import { observer } from 'mobx-react';
+import { IRootStore } from 'app/stores';
+import { componentInject } from 'app/shared/util/typed-inject';
+import { FaChevronRight, FaChevronLeft } from 'react-icons/fa';
+
+export interface IGenericSidebarProps extends StoreProps {
+  children: React.ReactNode;
+}
+
+const GenericSidebar = ({ children, showGenericSidebar, toggleGenericSidebar, genericSidebarWidth }: IGenericSidebarProps) => {
```

Comment:
Should we just call is Sidebar? Will it collide with components from other library?

Reply:
It will collide. Changing it to OncoKBSidebar as discussed yesterday